### DISCUSSION TITLE
Impelementing validation set

### DISCUSSION THREAD

**Discussion by afifhaziq** [ContinualAI/avalanche#1697] [Feature Request]
Hi,

As mentioned on the title, i think it would be very helpful if you can implementing a validation set on the experience. So the experience can produce three stream if specified (e.g. Train, test, validation) as standard deep learning training. I do belive validation set is important to observe overfitting and a more accurate implementation of early stopping. so instead of observing trainining loss, early stopping is implemented based on the validation loss. Thank you.
